tfjs-core/src/ops/browser.ts

```diff
@@ -371,4 +386,44 @@ export async function toPixels(
   return bytes;
 }
 
+/**
+ * Draws a `tf.Tensor` to a canvas.
+ *
+ * When the dtype of the input is 'float32', we assume values in the range
+ * [0-1]. Otherwise, when input is 'int32', we assume values in the range
+ * [0-255].
+ *
+ * @param image A rank-2 tensor with shape `[height, width]`, or a rank-3 tensor
+ * with shape `[height, width, depth]`. Rank-2 tensor would be drawn in
+ * grayscale. For rank-3 tensor, the depth must be 1, 3 or 4: if depth is 1, it
+ * would be drawn in grayscale; if the depth is 3, the three values of a pixel
+ * would be corresponding to R, G and B channels and alpha would be 1 (opaque)
+ * or customized in drawOptions; if the depth is 4, the four values of a pixel
+ * would be corresponding to R, G, B and A (alpha) channels.
+ * @param canvas The canvas to draw to.
+ * @param canvasOptions A object to configure the canvas to draw to.
+ * @param drawOptions A object of options to customize drawing.
```

Comment:
Should we describe the settings that people can change on these config and draw options here, or is this documented elsewhere?

Reply:
This is at the definition of `drawOptions` and `canvasOptions` and tfjs-website would add them to the API. Similar as `Options` of [this API](https://github.com/tensorflow/tfjs/blob/tfjs-v4.4.0/tfjs-converter/src/executor/graph_model.ts#L600-L618).